### ISSUE TITLE
test_get_ucx_version - RuntimeWarning: coroutine 'BlockingMode._arm_worker' was never awaited

### ISSUE THREAD

**Issue by jakirkham** [rapidsai/ucx-py#622]
Am seeing the following warning coming up when running the test suite. This comes up when running the `test_get_ucx_version` in `tests/test_version.py`. The warning is a bit unexpected as it wouldn't seem we would need `BlockingMode._arm_worker` here, but maybe I'm missing something.

```python
tests/test_version.py::test_get_ucx_version
  /datasets/jkirkham/devel/ucx-py/ucp/core.py:775: RuntimeWarning: coroutine 'BlockingMode._arm_worker' was never awaited
    gc.collect()
```

**Comment by pentschev** [comment]
That seems dangerous, at what point do you see this, any hints if it happens at the beginning, during transfers or when test is finalizing? Also is that message happening only once or multiple times?

I think this may have been introduced after 0.14, as I don't recall seeing such error back when I did some work on UCX-Py's tests.

**Comment by jakirkham** [comment]
Unfortunately I don't have a clear idea about when it happens. `pytest` just reports this after running the test suite. Though as it captures warnings and errors, that doesn't necessarily tell us when it happened. It seems to only happen once.

Looking at the test, it isn't really doing anything with UCX data transmission. So it's possible this is left over from an earlier test and we only hear about it here because of the `ucp.reset()` call.

https://github.com/rapidsai/ucx-py/blob/5652cb37063bd2f69020ffbd7f84e02a97fac267/tests/test_version.py#L4-L10

**Comment by jakirkham** [comment]
Yeah I think something else is causing this warning and we just happen to see it here. If I just run the tests, in `tests/test_version.py` no warning occurs; so, that can't be the true cause.

**Comment by pentschev** [comment]
This is fixed in UCXX via https://github.com/rapidsai/ucxx/pull/116. No new updates to UCX-Py will happen.
